<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCorpusItemsWizardCorrespondancesPage</class>
 <widget class="QWizardPage" name="ImportCorpusItemsWizardCorrespondancesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupSpeakerConvention">
     <property name="title">
      <string>Batch update</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTierNameFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tier name filter:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QLineEdit" name="editFilter"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelUpdateTo">
        <property name="text">
         <string>Update to:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelLevel">
        <property name="text">
         <string>Level:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="editLevel"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelAttribute">
        <property name="text">
         <string>Attribute:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="editAttribute"/>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="commandBatchUpdate">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupTiers">
     <property name="title">
      <string>Tier correspondances with Annotation Levels and Attributes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableView" name="tableviewTiers"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="optionCreateLevelsAttributes">
        <property name="text">
         <string>Create Annotation Levels / Attributes if they do not already exist in the corpus annotation structure.</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Encoding settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelEncoding">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select the encoding that will be used to read imported text files:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboEncoding"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboFilter</tabstop>
  <tabstop>editFilter</tabstop>
  <tabstop>editLevel</tabstop>
  <tabstop>editAttribute</tabstop>
  <tabstop>commandBatchUpdate</tabstop>
  <tabstop>tableviewTiers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
